komodo/switch.py

```diff
@@ -4,9 +4,9 @@
 from jinja2 import Template
 
 MIGRATION_WARNING = (
-    "Attention! Your machine is running on a RHEL6 environment "
-    "which is in the process of being phased out.\n"
-    "New versions of komodo will from October 2020 only support RHEL7.\n"
+    "Attention! Your machine is running on an environment "
+    "that is not supported. Rhel6 has been phased out.\n"
```

Comment:
Good idea. Suggest using RHEL6 (caps) as it's more readable. Also suggest removing dramatic exclamation point after Yammer.

Reply:
Thanks! Fixed :-)